sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -287,7 +287,7 @@ static RecognizeLinkedEntitiesResultCollection getExpectedBatchLinkedEntities()
         LinkedEntity linkedEntity = new LinkedEntity(
             "Seattle", new IterableStream<>(Collections.singletonList(linkedEntityMatch)),
             "en", "Seattle", "https://en.wikipedia.org/wiki/Seattle",
-            "Wikipedia");
+            "Wikipedia", "5fbba6b8-85e1-4d41-9444-d9055436e473");
```

Comment:
Is this the bingEntitySearchApiId ?
If yes, then why is the one on L302 different, or just test data?

Reply:
they are two difference bingId.